src/dtls13.c

```diff
@@ -1190,6 +1190,26 @@ int Dtls13ReconstructEpochNumber(WOLFSSL* ssl, byte epochBits,
     return SEQUENCE_ERROR;
 }
 
+int Dtls13GetUnifiedReaderSize(const byte input, word16* size)
+{
+    if (size == NULL)
+        return BAD_FUNC_ARG;
+
+    if (input & DTLS13_CID_BIT) {
+        WOLFSSL_MSG("DTLS1.3 header with connection ID. Not supported");
+        return WOLFSSL_NOT_IMPLEMENTED;
+    }
+
+    /* flags (1) + seq 8bit (1) */
+    *size = 2;
+    if (input & DTLS13_SEQ_LEN_BIT)
+        (*size)++;
+    if (input & DTLS13_LEN_BIT)
+        *size += 2;
```

Comment:
I would prefer to use `OPAQUE8_LEN` and `OPAQUE16_LEN` here. Like `*size = OPAQUE8_LEN + OPAQUE8_LEN;`